vta/src/runtime.cc

```diff
@@ -941,7 +941,9 @@ class CommandQueue {
     insn->memory_type = dst_memory_type;
     insn->sram_base = dst_sram_index;
     DataBuffer* src = DataBuffer::FromHandle(src_dram_addr);
-    insn->dram_base = src->phy_addr() / GetElemBytes(dst_memory_type) + src_elem_offset;
+    uint32_t elem_bytes = GetElemBytes(dst_memory_type);
+    CHECK_GE(VTA_PAGE_BYTES, elem_bytes);
```

Comment:
Perhaps it would be better to embed this check inside of the GetElemBytes() call insead since this gets called in a couple places and we probably want to make the page table check there. I'd also add a comment to describe what we are checking for.

Reply:
fixed.